app/views/shared/_button.html.erb

```diff
@@ -0,0 +1,33 @@
+<%# polaris button %>
+
+<div class="Polaris-ButtonGroup__Item">
+  <%# check passed variables %>
+  <% use_method = local_assigns[:method] || :get %>
+  <% use_style_class = '' %>    
+  <% use_style_class = ' ' + local_assigns[:style_class] if local_assigns[:style_class] %>    
+  <%# button %>
+  <%# disabled? %>
+  <% 
+    if local_assigns[:active_button] && local_assigns[:active_button] === false 
+      enabled = false
+    else
+      enabled = true
+    end
+  %>
```

Comment:
I'm not entirely sure I'm clear on what's happening in this section, but it _seems_ like a lot of logic to dump into a view. Typically if you're evaluating _something_ for the purposes of assigning a dynamic value into your views you'll want to extract this into a helper.

Reply:
Totally